### PR TITLE
added more user and server commands as per #85

### PR DESCRIPTION
**Pull request by diminDDL**
added more user and server commands as per #85

### CODE REVIEW

File: thatkitebot/cogs/utilitiescog.py [hunk 1]
```diff
@@ -64,6 +65,55 @@ async def status(self, ctx):
...
+        """
+        if user is None:
+            user = ctx.author
+        embed = discord.Embed()
+        embed.title = user.name
+        embed.description = user.mention
+        embed.add_field(name="ID", value=user.id)
+        embed.add_field(name="Server joined", value=user.joined_at.strftime("%d/%m/%Y %H:%M:%S"))
+        age = relativedelta(datetime.now(timezone.utc), user.joined_at)
+        embed.add_field(name="Server age", value=(f"{(str(age.years) + ' years, ' if age.years != 1 else ' year' + ', ') if age.years != 0 else ''}{(str(age.months) + ' months, ' if age.months != 1 else ' month, ') if age.months != 0 else ''}{age.days} {'days' if age.days != 1 else 'day'}"))
```
Comment (ThatRedKite) [inline thatkitebot/cogs/utilitiescog.py:104]: that is a bit too pythonic even for me :trollface:

File: thatkitebot/cogs/utilitiescog.py [hunk 2]
```diff
@@ -64,6 +65,55 @@ async def status(self, ctx):
...
+        embed = discord.Embed()
+        embed.title = guild.name
+        embed.description = guild.description
+        embed.add_field(name="Owner", value=guild.owner.mention)
+        embed.add_field(name="Members", value=guild.member_count)
+        embed.add_field(name="Channels", value=len(guild.channels))
+        embed.add_field(name="Roles", value=len(guild.roles))
+        embed.add_field(name="Created at", value=guild.created_at.strftime("%d/%m/%Y %H:%M:%S"))
+        age = relativedelta(datetime.now(timezone.utc), guild.created_at)
+        embed.add_field(name="Age", value=(f"{(str(age.years) + ' years, ' if age.years != 1 else ' year' + ', ') if age.years != 0 else ''}{(str(age.months) + ' months, ' if age.months != 1 else ' month, ') if age.months != 0 else ''}{age.days} {'days' if age.days != 1 else 'day'}"))
```
Comment (ThatRedKite) [inline thatkitebot/cogs/utilitiescog.py:84]: Mmmm Pythonic

**Review by ThatRedKite — CHANGES_REQUESTED** [review]
Maybe you could make them "user commands" that appear in a context menu?

File: thatkitebot/cogs/utilitiescog.py (hunk 1 above)
Comment (diminDDL) [reply thatkitebot/cogs/utilitiescog.py:104]: image

**Pull request merged**